### PR TITLE
Add support for single file hubs

### PR DESCRIPTION
**Pull request by rbuels**

### CODE REVIEW

File: src/raStanza.test.ts [hunk 1]
```diff
@@ -179,6 +179,7 @@ describe('RaStanza reader', () => {
     const input = 'key1 value1\nkey2 value2\nkey3\n'
     const stanza = new RaStanza(input)
     let updatedValue = stanza.get('key2')
+    assert(updatedValue)
```
Comment (cmdcolin) [inline src/raStanza.test.ts:182]: can we avoid using the 'assert' library?

**Comment by cmdcolin** [comment]
we don't use `assert` in any of our other libraries currently as far as I know, so it's a little out of place to me, and it seems like it can instead use a jest 'expect' type statement instead? it is also a node built-in, but it is being installed as an npm module https://www.npmjs.com/package/assert

**Comment by rbuels** [comment]
the assert is there pretty much just for TS.  jest expect isn't understood
by typescript to stop execution of the function like assert is, for
whatever reason.
I removed the dev dep on assert, is that ok?

**Comment by cmdcolin** [comment]
if you can add a comment saying why it is there (e.g. the typescript thing) it may help future readers, but otherwise it is fine

**Comment by cmdcolin** [comment]
let's get this one going :)

**Pull request merged**